Crm/Contact.php

```diff
@@ -701,6 +701,21 @@ public function toArrayForApi()
                 $return['custom_field_' . $fieldID] = $fieldValue;
             }
         }
+        if ($this->getBic()) {
+            $return['bic'] = $this->getBic();
```

Comment:
This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php